cmd/did-method-cli/sample/publickeys.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+  "id": "key1",
+  "type": "Ed25519VerificationKey2018",
+  "encoding": "Jwk",
+  "keyType": "Ed25519",
+  "purpose": ["verificationMethod"],
+  "value": "iZ+t8gA5DCW1P5bpVgHPIyk3QqzBOu9fohinLdkR4VE="
+ },
+ {
```

Comment:
we should manage recovery key differently, not like regular key to be added in the document

Reply:
done